debezium-platform/postgresql-kafka-example/README.md

```diff
@@ -0,0 +1,147 @@
+Using Debezium-platfrom to manage and stream changes
+===
+This example  will walk you thru on how to use the Debezium-platfrom-stage to manage and stream changes from a PostgreSQL database into Apache Kafka with Debezium Server deployed in a Kubernetes cluster.
+
+
+Preparing the Environment
+---
+As the first step we will provision a local Kubernetes cluster using [minikube](https://minikube.sigs.k8s.io/docs/) and will install an ingress controller. For this example, considering a local setup, we will use the `/etc/hosts` to resolve the domain.
+The following script, when executed, will use minikube to provision a local k8s cluster named `debezium` and will add the required ingress controllers. It will also updarte the `/etc/hosts` to add the domain url.
+
+```sh
+./create-environment.sh
+```
+> **_NOTE:_**
+If you are using minikube on Mac, you need also to run the `minikube tunnel -p debezium` command. For more details see [this](https://minikube.sigs.k8s.io/docs/drivers/docker/#known-issues) and [this](https://stackoverflow.com/questions/70961901/ingress-with-minikube-working-differently-on-mac-vs-ubuntu-when-to-set-etc-host).
+
+Now that you have the required k8s environment setup, its time to fire the required infra for this example. As we will be using PostgreSQL database and the Apache Kafka broker as ssource and the destination for our pipeline. The following script will create a dedicated namespace `debezium-platform` and use it going forward for further installations of our example. It will alos provision the both PostgreSQL database and the Apache Kafka broke
+
+```shell
+./setup-infra.sh
+```
+
+Deploying Debezium-platform Operator
```

Comment:
I will use the helm chart packaged in our chart repo. Please see [Installation](https://debezium.io/documentation/reference/3.1/operations/debezium-platform.html#_installation)

Reply:
sure, I was going to update it.